### PR TITLE
Update blueprint template with more specific sections and prompts

### PR DESCRIPTION
**Pull request by rawlinp**

### CODE REVIEW

**Comment by mitchell852** [comment]
man, you put a lot of thought into this. i like it.

File: blueprints/BLUEPRINT_TEMPLATE.md [hunk 1]
```diff
@@ -1,23 +1,220 @@
...
+<!--
+*How* will this impact the database schema?
+
+What new tables and columns will be required?
+How will existing tables and columns be changed?
+What are the column data types and modifiers?
+What are the FK references and constraints?
+-->
+
+ORT
```
Comment (mitchell852) [inline blueprints/BLUEPRINT_TEMPLATE.md]: change to `ORT Impact` for consistency?

**Comment by rawlinp** [comment]
Thanks, the blueprint idea has been on my mind for a bit :)

**Review by mitchell852 — COMMENTED** [review]
just one comment

File: blueprints/BLUEPRINT_TEMPLATE.md (hunk 1 above)
Comment (rawlinp) [reply blueprints/BLUEPRINT_TEMPLATE.md]: done

**Pull request merged**